<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1257</width>
    <height>810</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="2">
       <widget class="QPushButton" name="goto_boxes_btn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Boxes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="goto_search_btn">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="goto_vials_btn">
        <property name="text">
         <string>Vials</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="goto_microtubes_btn">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Microtubes</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>438</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="QTabWidget" name="plates_tab_wg">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="new_plates_tab">
         <attribute name="title">
          <string>New Plates</string>
         </attribute>
         <widget class="QPushButton" name="new_plates_save_btn">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>190</y>
            <width>121</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(51, 122, 183)</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>35</x>
            <y>30</y>
            <width>231</width>
            <height>150</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="text">
              <string>New Plates</string>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QSpinBox" name="new_n_plates_sb">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>1000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="new_plates_type_cb"/>
           </item>
           <item row="4" column="0">
            <widget class="QLineEdit" name="new_plate_comment_eb">
             <property name="maximumSize">
              <size>
               <width>320</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>Comment</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QLabel" name="new_plates_res_lab">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>250</y>
            <width>231</width>
            <height>371</height>
           </rect>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="edit_plates_tab">
         <attribute name="title">
          <string>Edit Plates</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="2" column="1" rowspan="3" colspan="5">
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="1" column="1">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>508</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0" rowspan="2">
             <widget class="QTableWidget" name="plate_table">
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>160</number>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <column>
               <property name="text">
                <string>Well</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Compound Id</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Batch</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Form</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Conc (mM)</string>
               </property>
              </column>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QToolButton" name="plate_export_btn">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="4">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTextEdit" name="plate_comment_tb">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>120</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>360</width>
                <height>120</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="plate_comment_btn">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(51, 122, 183)</string>
              </property>
              <property name="text">
               <string>Edit
Comment</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer_6">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="6">
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="5">
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>180</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>15</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1" colspan="2">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="0">
             <widget class="QLineEdit" name="plate_search_eb">
              <property name="maximumSize">
               <size>
                <width>360</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Plate</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>100</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="plate_search_btn">
              <property name="maximumSize">
               <size>
                <width>100</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(51, 122, 183)</string>
              </property>
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Search for Plate</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="upload_plates_tab">
         <attribute name="title">
          <string>Upload Plates from File</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <spacer name="verticalSpacer_8">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QGridLayout" name="gridLayout_9">
              <item row="2" column="1">
               <spacer name="horizontalSpacer_10">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Preferred</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>418</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="0">
               <widget class="QPushButton" name="choose_file_btn">
                <property name="text">
                 <string>Choose File</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="path_lab">
                <property name="text">
                 <string>No file chosen</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_12">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>248</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="upload_file_btn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(51, 122, 183)</string>
              </property>
              <property name="text">
               <string> Upload</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_11">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>17</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <widget class="QTableWidget" name="upload_plates_table">
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>140</number>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Plate Id</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Well</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Compound Id</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Batch</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Form</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Conc (mM)</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>16</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_9">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="goto_plates_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Plates</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="structure_lab">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(211, 215, 207); border: 1px solid black;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>428</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
